packages/@okta/vuepress-theme-prose/components/Sidebar.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+
+  <aside class="landing-navigation">
+    <ul class="landing">
+      <li :class="{overview: true}">
+          <router-link :to="overview.path">{{overview.title}}</router-link>
+        </li>
+        <li v-for="link in navigation" :key="link.title" :class="{subnav: showSublinks(link), open: $page.path.includes(link.path)}">
+          <div class="link-wrap">
+            <svg viewBox="0 0 320 512" v-if="showSublinks(link) && !$page.path.includes(link.path)">
+              <path d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"/>
+            </svg>
+
+            <svg viewBox="0 0 320 512" v-if="showSublinks(link) && $page.path.includes(link.path)" class="open">
+              <path d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z"/>
+            </svg>
+
+
+            <router-link :to="link.path">{{link.title}}</router-link>
+          </div>
+          
+          <ol v-if="subLinks(link) && $page.path.includes(link.path)"  class="sections">
+            <li v-for="subLink in subLinks(link)" :key="subLink.title">
+              <router-link :to="subLink.path" :class="{active: isActive(subLink)}">{{subLink.title}}</router-link>
+            </li>
+          </ol>
+        </li>
+    </ul>
+  </aside>
+  
+</template>
+
+<script>
+  import _ from 'lodash';
+  export default {
+    name: 'Sidebar',
+    props: {
+      oldNavigation: {
+        default: true,
+        type: Boolean
+      }
+    },
+    data () {
```

Comment:
nit: space between data and () is confusing and inconsistent (see `navigation()` below)